CMakeLists.txt

```diff
@@ -101,9 +102,19 @@ target_link_objects(
   irods_s3_api_core
   irods_s3_api_endpoint_authentication
   irods_s3_api_endpoint_put_object
+  irods_s3_endpoints
 )
 
-install(TARGETS ${IRODS_S3_API_BINARY_NAME} DESTINATION "${CMAKE_INSTALL_BINDIR}")
+target_include_directories(
+  ${IRODS_S3_API_BINARY_NAME}
+  PRIVATE
+  third-party/hmac_sha256
+)
```

Comment:
I don't think this call is needed. This target is made of object libraries and has no source files, so doesn't need include directories.

Reply:
Removed these lines.